main.js

```diff
@@ -0,0 +1,48 @@
+function generateNumbers() {
+    let numbersOnScreen = Math.floor(Math.random() * 3) + 3;
+    let allNumbers = [];
+
+    for (let i = 0; i < numbersOnScreen; i++) {
+        let randNumber = Math.floor(Math.random() * 101);
+
+        allNumbers.push(randNumber);
+    }
+
+    return allNumbers;
+}
+
+function displayNumbers(allNumbers) {
+    let numbersOnScreen = allNumbers.length;
+
+     // using a for loop instead of forEach as the array.pop changes the length over each iteration
+     for (let i = 0; i < numbersOnScreen; i++) {
+        let randNumber = allNumbers.pop();
+        let number = document.createElement('div');
+
+        number.textContent = randNumber;
+        number.setAttribute('class', 'numberBox');
+        number.style.height = '100px';
+        number.style.width = '100px';
+        number.style.display = 'flex';
+        number.style.alignItems = 'center';
+        number.style.justifyContent = 'center';
```

Comment:
Don't add this with JavaScript, just apply a class with this all predefined in a CSS class.